src/Carnac/UI/PreferencesViewModel.cs

```diff
@@ -65,7 +82,6 @@ public string Version
                                                        {
                                                          "MahApps.Metro",
                                                          "Fody",
-                                                         "Caliburn Micro",
```

Comment:
You should add `Shouldly` in here, given you've mentioned test frameworks below...